plugins/qcheck-stm/src/ortac_qcheck_stm.ml

```diff
@@ -4,9 +4,12 @@ module Ir_of_gospel = Ir_of_gospel
 module Reserr = Reserr
 module Stm_of_ir = Stm_of_ir
 
-let main path init sut () =
+let main path init sut output () =
   let open Reserr in
-  let pp = Fmt.((pp Ppxlib_ast.Pprintast.structure) stdout) in
+  let channel =
+    Registration.get_channel output |> Format.formatter_of_out_channel
+  in
+  let pp = (pp Ppxlib_ast.Pprintast.structure) channel in
```

Comment:
Why the extra parentheses? (they were there before, I know, but still)

Reply:
No reason, I've removed them.